cfme/utils/bz.py

```diff
@@ -90,26 +92,27 @@ def from_config(cls):
                    password=credentials.get(cred_key, {}).get("password"),
                    cookiefile=None,
                    tokenfile=None,
-                   product=bz_conf.get("bugzilla", {}).get("product"))
+                   product=bz_conf.get("bugzilla", {}).get("product"),
+                   config_options=bz_conf)
 
     @cached_property
     def bugzilla(self):
         return _Bugzilla(**self.__kwargs)
 
     @cached_property
     def loose(self):
-        return cfme_data.get("bugzilla", {}).get("loose", [])
+        return self.config_options.get("loose", [])
 
     @cached_property
     def open_states(self):
-        return cfme_data.get("bugzilla", {}).get("skip", set([]))
+        return self.config_options.get("skip", set([]))
```

Comment:
what is [] for ?

Reply:
its not needed anymore - but it was comonly needed on old python